.github/scripts/find-ta-tasks.sh

```diff
@@ -0,0 +1,30 @@
+#!/usr/bin/env bash
+
+SCRIPT_DIR=$( cd -- "$( dirname -- "${BASH_SOURCE[0]}" )" &> /dev/null && pwd )
+
+compatibleTrustedArtifactsBasedTasks=()
+nonCompatibleTrustedArtifactsBasedTasks=()
+for dir in $(find . ${SCRIPT_DIR}/../../tasks/managed/ -maxdepth 1 -type d | sort)
+do
+  baseDir=$(basename "${dir}")
+  taskFile="${dir}/${baseDir}.yaml"
+  if [ -f "${taskFile}" ]; then
+    ociStageParam=$(yq '.spec.params[] | select(.name == "ociStorage") | .name' "${taskFile}")
+    if [ ! -z "${ociStageParam}" ]; then
+      echo "+ - $(basename ${dir})"
+      compatibleTrustedArtifactsBasedTasks+=(tasks/managed/$(basename ${dir}))
+    else
+      echo "x - $(basename ${dir})"
+      nonCompatibleTrustedArtifactsBasedTasks+=(tasks/managed/$(basename ${dir}))
+    fi
+  fi
+done
+
+echo ""
```

Comment:
why not a single 
`echo "
BLA

BLA"`

Reply:
sure. I can change if you feel strongly about it.

Reply:
IMHO, this fells like a "dripping tap". For a single case, it does not represent much, but a full cluster wasting resources repeating commands that consumes (even a bit) of cpu clocks/pids/fd  for  something that could be done once.. not good - again, IMHO - if you don't feel like changing ti, NP.

Reply:
this script is actually meant to be run on your laptop...as a means to see which tasks have not been converted yet. It is NOT run as part of a pipeline.